packages/hardhat-core/test/internal/hardhat-network/helpers/assertions.ts

```diff
@@ -54,7 +54,10 @@ export async function assertProviderError(
     }
 
     if (message !== undefined) {
-      assert.include(error.message, message);
+      assert.include(
```

Comment:
@fvictorio do we provide guarantees for error messages w.r.t capitalisation of characters?

Reply:
I changed the first letter of a sentence to be capitalised